File: zc1/OverloadControl/OverloadControl.Models/Case.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OverloadControl.Models
{
    /// <summary>
    /// 案件表
    /// </summary>
   public class Case
    {
        public string CaseNo { get; set; }
        public int Id 
        { get; set; }
        public string Address 
        { get; set; }
        public string Types 
        { get; set; }
        public string Details 
        { get; set; }
        public string Platenumber
        { get; set; }
        public DateTime Date
        { get; set; }
        public string Phone
        { get; set; }
        public string Image
        { get; set; }
        public string PolicerName
        { get; set; }
        public string State
        { get; set; }
        public string ViolatorsName
        { get; set; }
        public string ApplicationTime
        { get; set; }
        public string Description
        { get; set; }
        public string Model
        { get; set; }
        public string Judgment
        { get; set; }
        public string M_Content
        { get; set; }
        public string Content
        { get; set; }
        public string PolicerName1
        { get; set; }
        public string ViolatorsPhone
        { get; set; }
        public string OrderTake { get; set; }
        public string LegalArticles{ get; set; }

        public List<Case_Progress> Case_Progresses { get; set; } 
        public List<Law_Case> law_Cases { get; set; } 
        public List<Police_Case> Police_Cases { get; set; } 
        //public int VId { get; set; }
        //public Violators Violators { get; set; }
    }
}

File: 案件处理人员界面/src/type/case.ts


export interface ListInt {
  CaseNo: string;
Id: number;
Address: string;
Types: string;
Details: string;
Platenumber: string;
Date: Date;
Phone: string;
Image: string;
PolicerName: string;
State: string;
ViolatorsName: string;
ApplicationTime: string;
Description: string;
Model: string;
Judgment: string;
M_Content: string;
Content: string;
PolicerName1: string;
ViolatorsPhone: string;
OrderTake: string;
LegalArticles: string;
LawTypeId:number
}
interface SelectDataInt {
  status: string;
  page: number; //页码
  count: number; //总条数
  pagesize: number; //默认一页显示几条
}
interface ActiveInt {
   CaseNo: string;
Id: number;
Address: string;
Types: string;
Details: string;
Platenumber: string;
Date: Date;
Phone: string;
Image: string;
PolicerName: string;
State: string;
ViolatorsName: string;
ApplicationTime: string;
Description: string;
Model: string;
Judgment: string;
M_Content: string;
Content: string;
PolicerName1: string;
ViolatorsPhone: string;
OrderTake: string;
LegalArticles: string;
LawTypeId:number
}
interface ActiveInt2{
  Id: number,
  Content:string
}
interface LawInt{
  Id: number,
  Content:string
}
export class InitData {
  lawData: LawInt[] = []
  active2: ActiveInt2 = {
    Id: 0,
    Content: ""
  }
  selectData: SelectDataInt = {
    status: "",
    page: 1,
    count: 0,
    pagesize: 5,
  };
  list: ListInt[] = []; //用来接受用户信息的列表
  isShow = false;
  isAddShow = false;
  isCancelShow = false;
  isCheck2 = false;
  isRollBackShow = false;
  active: ActiveInt = {
   CaseNo:"",
    Id: 0,
  Address: "",
  Types: "",
  Details: "",
  Platenumber: "",
  Date: new Date(),
  Phone: "",
  Image: "",
  PolicerName: "",
  State: "",
  ViolatorsName: "",
  ApplicationTime: "",
  Description: "",
  Model: "",
  Judgment: "",
  M_Content: "",
  Content: "",
  PolicerName1: "",
  ViolatorsPhone: "",
  OrderTake: "",
  LegalArticles: "",
  LawTypeId:0
  };
}
